connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -0,0 +1,51 @@
+package com.theoplayer.android.connector.uplynk
+
+import com.theoplayer.android.api.THEOplayerView
+import com.theoplayer.android.api.ads.Ad
+import com.theoplayer.android.api.ads.ServerSideAdIntegrationController
+import com.theoplayer.android.api.source.ssai.CustomSsaiDescriptionRegistry
+import com.theoplayer.android.connector.uplynk.internal.UplynkAdIntegration
+import com.theoplayer.android.connector.uplynk.internal.UplynkSsaiDescriptionConverter
+
+internal const val TAG = "UplynkConnector"
+
+/**
+ * A connector for the Uplynk Media Platform.
+ *
+ * @param theoplayerView
+ *   The THEOplayer view, which will be connected to the created connector.
+ *
+ *   @see [Uplynk Media Platform](https://docs.edgecast.com/video/index.html)
```

Comment:
Nit-pick: fix spacing.
```suggestion
 * @see [Uplynk Media Platform](https://docs.edgecast.com/video/index.html)
```